package.json

```diff
@@ -113,8 +129,8 @@
     "lint": "eslint src",
     "lint-fix": "eslint --fix src",
     "precommit": "lint-staged && npm run test",
-    "prefast-test": "npm run prepublish",
-    "prepublish": "npm run package",
+    "prefast-test": "npm run prepublishOnly",
+    "prepublishOnly": "npm run package",
```

Comment:
unrelated?

Reply:
It suppresses a warning about feature deprecation in new versions of node. Not unrelated because the packages and the newer node version (12.4.0) was tested with this update.